server/di-container.ts

```diff
@@ -20,12 +20,28 @@ import {
   validateReplyPUT
 } from './middleware/validation';
 
+// Either use local DB with dev mode on typescript or remote DB in prod mode on server with environment variables
+const path = process.env.ENTITIES || './server/entities/*.ts';
+const connection: any = {
+  type: 'mariadb',
+  url:
+    process.env.JAWSDB_MARIA_URL ||
+    'mysql://BulletinBoarder:fjWvxuY4edDwGiAv6qOa6@localhost:3306/relaa',
```

Comment:
Now since these defaults don't make sense to all team members it would be good idea not to have these defaults which are unique for each environment.

You could enforce this by saying in README what new developer have to do in order to get this project setup with environment variables :)

Reply:
I was wondering about the solution for this problem while making it and playing around with .env files just seemed too sketchy to me knowing that the project will get dockerized in the future where you can set the environment as you like!

However as I still felt this could potentially be an issue which I would face again often in the future, I wanted to put lots of time and effort for solving this issue with the most efficient way possible! Turns out dotenv.config() has a built in silent failure mechanism which made it so that no extra hastle was even needed! Now the db configuring has been moved to its own file and instructions for filling the .env has been added to README!